pumps/mongo.go

```diff
@@ -297,17 +297,22 @@ func (m *MongoPump) WriteData(data []interface{}) error {
 
 			analyticsCollection := sess.DB("").C(collectionName)
 
+			nonce := time.Now() //needed in case the flush to the log is out of sync
 			log.WithFields(logrus.Fields{
 				"prefix": mongoPrefix,
-			}).Info("Purging ", len(dataSet), " records")
+			}).Info("Purging ", len(dataSet), " chunks (for nonce: ", nonce, ").")
```

Comment:
I think this is wrong. They are `len(dataSet)` records but just `1` chunk. So it should stay same. Also, it will log the the time, why do we add time again ?